wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -614,62 +614,6 @@ private void checkAlerts() {
         url.addEncodedQueryParameter("package", packageInfo.packageName);
         url.addQueryParameter("current", Integer.toString(packageInfo.versionCode));
 
-		/*new HttpGetThread(url.build(), application.httpUserAgent()) {
```

Comment:
This has been commented out for 8 years. Probably safe to remove.

Reply:
Bitcoin Wallet uses this method to determine if there is a new version published to Google Play.  If so, the app can remind to upgrade using Google Play or perhaps Fdroid depending on which store they were using.

We never used this mechanism.